charts/sourcegraph/Chart.yaml

```diff
@@ -5,7 +5,7 @@ description: Chart for installing Sourcegraph
 type: application
 
 # Chart version, separate from Sourcegraph
-version: 0.3.0
+version: 1.0.0
```

Comment:
I'm not sure we're ready to 1.0 yet! Maybe just bump to `0.4.0` and we can 1.0 as part of going GA?

Reply:
I know that's not following strict semver, but 1.0 is special 😄